WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -401,8 +401,8 @@ <h2 class="text-center text-lowercase mb-4">Frequently Asked Questions</h2>
                 </div>
 
                 <div class="card bg-white box-shadow border-light mb-2">
-                    <input type="checkbox" id="faq00" class="toggle-input" />
-                    <label class="card-header bg-light font-weight-bold text-uppercase border-0 toggle-label pr-3" for="faq00">
+                    <input type="checkbox" id="faq7" class="toggle-input" />
```

Comment:
I am not familiar with this notation. Why was it faq00 before?

Reply:
Me neither. I guess it was just a mistake, @mayankchhabra can confirm that.

Reply:
@yahiheb you're right, it was indeed a mistake. Good find!